backend/config/database/games.ts

```
@@ -0,0 +1,266 @@
+  if (path.startsWith('/') || path.endsWith('/')) {
+    return false;
+  }
+  
+  // Prevent encoded characters that could bypass validation
+  if (path.includes('%') || path.includes('\\u') || path.includes('\\x')) {
+    return false;
+  }
+  
+  // Prevent null bytes and other control characters
+  // eslint-disable-next-line no-control-regex
+  if (path.includes('\0') || /[\x00-\x1f\x7f-\x9f]/.test(path)) {
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)